apps/articles-web-template/env.mjs

```
@@ -0,0 +1,22 @@
+import { createEnv } from '@t3-oss/env-nextjs';
+import { z } from 'zod';
+
+export const env = createEnv({
+  client: {
+    NEXT_PUBLIC_API_BASE: z.string().url(),
```

Biome (1.9.4)

[error] This property value named NEXT_PUBLIC_API_BASE is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named NEXT_PUBLIC_API_BASE


(lint/suspicious/noDuplicateObjectKeys)